com.gamestarmechanic.app.yaml

```diff
@@ -0,0 +1,77 @@
+---
+app-id: com.gamestarmechanic.app
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: gamestarmechanic-wrapper
+separate-locales: false
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+  
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --env=GAMESTAR_PLUGINS_PATH=/app/extra/plugins
+
+modules:
+  - name: gamestar-mechanic
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/gamestar-mechanic/flatpak-node/cache
+    sources:
+      - type: git
+        url: https://github.com/dylanmccall/gamestar-mechanic-linux.git
+        branch: main
```

Comment:
NEVER a git branch.

Reply:
Oh, embarrassing! I forgot to push all of my recent changes there, but indeed I did forget to remove the `branch` part :b Importantly, the url also changes to https://github.com/endlessm/gamestar-mechanic-linux.git in the newest commit.